rasa_core/events/__init__.py

```diff
@@ -98,6 +99,7 @@ def __init__(self, text, intent=None, entities=None, parse_data=None):
         self.text = text
         self.intent = intent if intent else {}
         self.entities = entities if entities else []
+        self.time = time.time()
```

Comment:
I think we talked about this already, but whats the reason to not do this at the event level, so for every event?

Reply:
yeah that would be better.

maybe: define an `__init__` method for the `Event` class and then in each of the subclasses, at the end of `__init__` call `super().__init__()` ?

Reply:
yes